app/internal/security/schema.py

```diff
@@ -15,3 +15,57 @@ class LoginUser(BaseModel):
 
     class Config:
         orm_mode = True
+
+
+class ForgotPassword(BaseModel):
+    """
+    BaseModel for collecting and verifying user
+    details sending a token via email
+    """
+    username: str
+    email: str
+    user_id: Optional[str] = None
+    token: Optional[str] = None
+    is_manager: Optional[bool] = False
+
+    class Config:
+        orm_mode = True
```

Comment:
Consider adding example - https://fastapi.tiangolo.com/tutorial/schema-extra-example/

Reply:
Can you please elaborate how will it contribute to the code functionality?

Reply:
For easier debugging, and for readability for other developers (for examples - in #docs views it's more readable)

Reply:
For my opinion, debugging is good enough just by using pydantic the way i did, and it's readable enough for any developer who knows how to work with pydantic models. Also, i added docstring. Thank You for your review.